### ISSUE TITLE
version 4.0.2 unfortunately is not as usable as the previous

### ISSUE THREAD

**Issue by vmgupta10gh** [ahmadnassri/app-restconsole#49]
- 4.0.2 requires more scrolling
- 4.0.2 does not support "tabbing" into the next field as all fields are locked by default
- need an option to hide unnecessary fields (there already is an option for this but not substantial) - i usually use only a few but still have to scroll up and down several times
- inspecting the response also requires more clicks - need to click for response body, need click for response headers etc. showing all the information (or having a default tab "All info") might not be a bad idea.
- why not provide an extremely basic just-a-vertical-list-of-fields type of theme for those of us who value efficiency over good-looking skins?

Keep up the good work though. This really is a one of a kind plugin.  Thanks!

**Comment by ahmadnassri** [comment]
Thanks for the feedback!

I'd start by saying the reason I changed the design is to make it more adaptive and be able to add more of the planned features that the old design was limiting. it has little to do with "good looking" or not, this time around I used Twitter's Bootstrap framework, which made it look pretty, but that wasn't my main intention ...

I agree with you, I hate scrolling, and I did spend a lot of time making the old version tab friendly! but very few people seemed to care or notice, so I put that as a low priority for this version ... but I'm glad you're bring it up, means somebody out there prefers tabbing like me :)

I will be adding the option to hide fields for sure, I wanted to do that, but wasn't enough time for v4.0.2

I probably wont do multiple skins, but will provide multiple viewing options that will make the experience better, your constant feedback helps :)

**Comment by intel352** [comment]
agreed with both of you, tabbing between fields & sections is preferred. Additionally, collapsing unused sections would be nice (at least with a manual toggle collapse).
